src/core/layertree/qgslayertreemodellegendnode.h

```diff
@@ -26,7 +26,10 @@
 #include "qgis_core.h"
 #include "qgis_sip.h"
 
+#include "qgsexpression.h"
```

Comment:
Is it possible to avoid this include? This file changes often, so reducing it's include dependency will help keeping the build time low.

Reply:
I will test to see if it build without

Reply:
yes, seems fine without it.